src/vorta/views/archive_tab.py

```diff
@@ -353,7 +355,8 @@ def on_selection_change(self, selected=None, deselected=None):
 
         if self.repoactions_enabled and len(indexes) == 1:
             # Enable archive actions
-            self.fArchiveActions.setEnabled(True)
+            for widget in [self.bMountArchive, self.bExtract, self.bRename]:
```

Comment:
I think this is a much nicer approach :)

One small thing: How about you extract `[self.bMountArchive, self.bExtract, self.bRename]` here and on line 371 into a variable with a descriptive name?

Reply:
Good idea!